### PR TITLE
feat: add basic page for riseupvpn

### PR DESCRIPTION
**Pull request by bassosimone**
Part of https://github.com/ooni/backend/issues/473

Next step: computing if there are errors.

### CODE REVIEW

File: components/measurement/nettests/RiseupVPN.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+
+const RiseupVPNDetails = ({ measurement, render }) => {
+  // TODO(bassosimone,cyberta): possibly refine the algorithm
```
Comment (bassosimone) [inline components/measurement/nettests/RiseupVPN.js:4]: @cyberta: does this look good to you as a first stab at scoring the measurement?

**Review by sarathms — APPROVED** [review]
This looks good. :whale: 
The only thing missing is the metadata for the new test (name, group, url) in `components/test-info.js`. I will add it because it needs some work on the copy as well.

**Comment by sarathms** [comment]
The logic to use a default blank measurement page with just the raw data was broken. I added a fix. With this, any measurement from tests that explorer doesn't recognize yet will still be shown like this:

image

**Comment by cyBerta** [comment]
Nice! I didn't follow this development the last days and I'm very happy to see this progress!

**Pull request merged**